src/App.js

```diff
@@ -1,11 +1,30 @@
+import { NavLink, Route, Routes } from 'react-router-dom';
 import Calculator from './components/Calculator';
 import ShowQuote from './components/ShowQuote';
+import Home from './components/Home';
 
 function App() {
   return (
     <div className="container">
-      <Calculator />
-      <ShowQuote />
+      <nav>
+        <h1>Math Magicians</h1>
+        <ul>
+          <li>
+            <NavLink to="/">Home</NavLink>
+          </li>
+          <li>
+            <NavLink to="/calculator">Calculator</NavLink>
```

Comment:
- [ ] Great job implementing the full website. Kindly ensure your design matches the given wireframe for the calculator page by adding  the text : `Lets do some math!` to the left and position the calculator component to the right as is in the given wireframe.

Wireframe
![image](https://user-images.githubusercontent.com/81932478/231312647-9c60dd40-4422-4425-9b43-bda3307da15e.png)

Your design
![image](https://user-images.githubusercontent.com/81932478/231312712-d47d8afc-0012-4e24-9d5c-0422ff77bbbf.png)